cartofacade/capi_test.go

```diff
@@ -161,23 +152,16 @@ func TestCGoAPI(t *testing.T) {
 		holder, err := vc.getPosition()
 
 		test.That(t, err, test.ShouldBeNil)
-		test.That(t, holder.ComponentReference, test.ShouldEqual, "C++ component reference")
-
-		test.That(t, holder.X, test.ShouldEqual, 100)
-		test.That(t, holder.X, test.ShouldEqual, 100)
-		test.That(t, holder.Y, test.ShouldEqual, 200)
-		test.That(t, holder.Z, test.ShouldEqual, 300)
+		test.That(t, holder.ComponentReference, test.ShouldEqual, "mysensor")
 
-		test.That(t, holder.Ox, test.ShouldEqual, 400)
-		test.That(t, holder.Oy, test.ShouldEqual, 500)
-		test.That(t, holder.Oz, test.ShouldEqual, 600)
+		test.That(t, holder.X, test.ShouldEqual, 0)
+		test.That(t, holder.Y, test.ShouldEqual, 0)
+		test.That(t, holder.Z, test.ShouldEqual, 0)
 
-		test.That(t, holder.Imag, test.ShouldEqual, 700)
-		test.That(t, holder.Jmag, test.ShouldEqual, 800)
-		test.That(t, holder.Kmag, test.ShouldEqual, 900)
-
-		test.That(t, holder.Theta, test.ShouldEqual, 1000)
-		test.That(t, holder.Real, test.ShouldEqual, 1100)
+		test.That(t, holder.Imag, test.ShouldEqual, 0)
+		test.That(t, holder.Jmag, test.ShouldEqual, 0)
+		test.That(t, holder.Kmag, test.ShouldEqual, 0)
+		test.That(t, holder.Real, test.ShouldEqual, 1)
```

Comment:
Why do these tests still work now that you are returning the real position?

Reply:
B/c it takes 3 lidar readings before GetPosition returns a non 0 result, see https://github.com/viamrobotics/viam-cartographer/pull/160/files#diff-0bd1cb9b5092e5c3c4e4ffa3be356ac99759fdc0c92546d1d7de11aed037dde0R602

Reply:
ohhhh I see, thanks!